frontend/src/components/AboutMe/ProfileCard.jsx

```
@@ -17,33 +21,127 @@
+        }
+      );
 
+      setUser(prev => ({
+        ...prev,
+        profile_picture: response.data.filename,
+      }));
+      setShowEditPopup(false);
+      setSelectedImage(null);
+      setPreviewURL(null);
+    } catch (err) {
+      alert("Upload failed: " + (err.response?.data?.detail || "Unknown error"));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)